src/components/desktop/SidePaneSwitch.js

```diff
@@ -0,0 +1,17 @@
+import { Route, Switch } from 'react-router-dom'
+
+import EntryPane from './EntryPane'
+import MainPane from './MainPane'
+
+const SidePaneSwitch = () => (
+  <Switch>
+    <Route path="/entry/:id">
+      <EntryPane />
+    </Route>
+    <Route>
```

Comment:
It seems the main page catches all other routes (acts as a default case when previous routes fall through). Could we enforce an exact blank route (`/`) here and instead default to a redirect to the blank route?

Reply:
This route should catch /map and /list without redirecting. You're right, we may want to explicitly have a route for each of those, with MainPane.